<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserisciNuovoLibro</class>
 <widget class="QDialog" name="inserisciNuovoLibro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreriaDigitale</string>
  </property>
  <widget class="QTextEdit" name="titoloLTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="dataLTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="autoreLTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="casaEditriceLTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="numPagineLTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="titoloLLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Titolo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dataLLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Data:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="autoreLLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Autore:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="casaEditriceLLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Casa editrice:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="numPagineLLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Numero pagine:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciNuovoLibroButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius: 8px;
background-color:#333333; 
border: none;
color: white;
padding: 2px;
text-align: center;
font-size: 15px;
font-family: &quot;Garamond&quot;;
}

:hover {
background-color: #676767;
}

:pressed{
border: 1px solid #E0E2E2;
}</string>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="libroInseritoTextEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 10pt;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="libroInseritoLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>220</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Il seguente libro Ã¨ stato correttamente aggiunto alla libreria:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
